### ISSUE TITLE
Multi-Instances not working

### ISSUE THREAD

**Issue by UnChatWTF** [VastraKai/McPatch#2]
After enabling the multi-instances option, the app manifest was invalid and couldn't find my game anymore.
![image](https://github.com/user-attachments/assets/d363f48e-ba29-4fe3-a562-ee5ecd112cf3)
![image](https://github.com/user-attachments/assets/568ef88e-4b2f-4dbe-9656-fcc17d2f243f)

**Comment by VastraKai** [comment]
yeah i'm not entirely concerned about this, i'm planning to rewrite McPatch soon along with how all of the patches work, just give me some time as I'm very busy
